src/Database/Relations/BelongsToMany.php

```diff
@@ -133,17 +135,14 @@ public function attach($id, array $attributes = [], $touch = true)
          *     $model->bindEvent('model.relation.beforeAttach', function (string $relationName, array $attachedIdList, array $insertData) use (\October\Rain\Database\Model $model) {
          *         if (!$model->isRelationValid($attachedIdList)) {
          *             throw new \Exception("Invalid relation!");
-         *             return false;
          *         }
          *     });
          *
          */
-        if ($this->parent->fireEvent('model.relation.beforeAttach', [$this->relationName, $attachedIdList, $insertData], true) === false) {
-            return;
-        }
+        $this->parent->fireEvent('model.relation.beforeAttach', [$this->relationName, $attachedIdList, $insertData]);
 
-        /**
-         * @see Illuminate\Database\Eloquent\Relations\Concerns\InteractsWithPivotTable
+        /*
+         * @See Illuminate\Database\Eloquent\Relations\Concerns\InteractsWithPivotTable
```

Comment:
@mjauvin Wasn't the original change correct?
```suggestion
        /**
         * @see \Illuminate\Database\Eloquent\Relations\Concerns\InteractsWithPivotTable
```